server/lease/lessor_test.go

```diff
@@ -207,6 +207,29 @@ func TestLessorRevoke(t *testing.T) {
 	}
 }
 
+func renew(t *testing.T, le *lessor, id LeaseID) int64 {
+	ch := make(chan int64, 1)
+	errch := make(chan error, 1)
+	go func() {
+		ttl, err := le.Renew(id)
+		if err != nil {
+			errch <- err
+		} else {
+			ch <- ttl
+		}
+	}()
+
+	select {
+	case ttl := <-ch:
+		return ttl
+	case err := <-errch:
+		t.Fatalf("failed to renew lease (%v)", err)
+	case <-time.After(2 * time.Second):
```

Comment:
Would you mind to increase the number into 30s or 1m? since the CI instance has limitted resource, it might be flaky case. The large number won't hurt the case if it is supposed to be fast. WDYT?

Reply:
It's just a unit test case, and there isn't too many lease in both cases, 30s timeout seems to be overkill. 2s should be OK. But it isn't a big deal, if you want, any value between 2 ~ 10s is OK to me.

Reply:
I increase the timeout to 10s. :slightly_smiling_face: